test/performance/odp_bench_packet.c

```diff
@@ -1840,6 +1840,7 @@ int main(int argc, char *argv[])
 	odp_pool_param_init(&params);
 	params.pkt.seg_len = PKT_POOL_SEG_LEN;
 	params.pkt.len     = TEST_MAX_PKT_SIZE;
+	params.pkt.max_len = 2 * TEST_MAX_PKT_SIZE;
```

Comment:
A problem with this solution is that the API promises only that the pool is able to allocate at least one `max_len` packet. In case of "bench_packet_ref_pkt" test, TEST_REPEAT_COUNT (=1000) packets are required.

A cleaner solution could be to use a larger pool, where `params.pkt.len = 2 * TEST_MAX_PKT_SIZE`, for these problematic tests. Another option would be to make the original packets smaller (`TEST_MAX_PKT_SIZE  / 2`).

Reply:
Thanks for the review comment, I will send V2 with "params.pkt.len = 2 * TEST_MAX_PKT_SIZE" change, so as to create a bigger pool.